src/sinks/datadog/logs/tests.rs

```diff
@@ -60,6 +59,11 @@ async fn start_test(
     let (expected, events) = random_lines_with_stream(100, 10, Some(batch));
 
     let _ = sink.run(events).await.unwrap();
+    // It's possible that the internal machinery of the sink is still
+    // spinning up. We pause here to give the batch time to wind
+    // through. Waiting is preferable to adding synchronization into the
+    // actual sync code for the sole benefit of these tests.
+    tokio::time::sleep(std::time::Duration::from_millis(500)).await;
 
     assert_eq!(receiver.try_recv(), Ok(batch_status));
```

Comment:
```suggestion
    assert_eq!(receiver.await, batch_status);

Reply:
Haha of course. There's s3 tests that need to be updated along these same lines.

Reply:
Resolved in 6368d9084